From 036bf473b96b551902621ec4b75898843c5d9b69 Mon Sep 17 00:00:00 2001
From: Maximiliano Sandoval <msandova@gnome.org>
Date: Thu, 21 Mar 2024 22:56:35 +0100
Subject: [PATCH] appdata: Remove redefined translate url

Fixes the warning:

W: org.gnome.TextEditor.desktop:25: url-redefined translate
   An URL of this type has already been defined.
---
 data/org.gnome.TextEditor.appdata.xml.in.in | 1 -
 1 file changed, 1 deletion(-)

diff --git a/data/org.gnome.TextEditor.appdata.xml.in.in b/data/org.gnome.TextEditor.appdata.xml.in.in
index 722455c..a7a1749 100644
--- a/data/org.gnome.TextEditor.appdata.xml.in.in
+++ b/data/org.gnome.TextEditor.appdata.xml.in.in
@@ -19,7 +19,6 @@
   <url type="homepage">https://apps.gnome.org/TextEditor/</url>
   <url type="bugtracker">https://gitlab.gnome.org/GNOME/gnome-text-editor/issues</url>
   <url type="donation">https://www.gnome.org/donate/</url>
-  <url type="translate">https://wiki.gnome.org/TranslationProject</url>
   <url type="vcs-browser">https://gitlab.gnome.org/GNOME/gnome-text-editor</url>
   <url type="contribute">https://welcome.gnome.org/app/TextEditor/</url>
   <url type="translate">https://l10n.gnome.org/module/gnome-text-editor/</url>
-- 
2.44.0

